openstates/in/committees.py

```diff
@@ -0,0 +1,115 @@
+
+import lxml.html
+
+from billy.scrape.committees import CommitteeScraper, Committee
+from apiclient import ApiClient
+from .utils import get_with_increasing_timeout
+
+class INCommitteeScraper(CommitteeScraper):
+    jurisdiction = 'in'
+
+    def process_special_members(self,comm,comm_json,role_name):
+        try:
+            mem = comm_json[role_name]
+        except KeyError:
+            return
+        if mem:
+            person = mem["firstName"]+" "+mem["lastName"]
+            comm.add_member(person,role=role_name)
```

Comment:
The `role`s are all spelled out (eg, `Ranking Minority Member` instead of `rankingMinMember`) on the committees' webpages (eg, `http://iga.in.gov/legislative/2015/committees/rules_and_legislative_procedures_1800`); we should use the proper names (`Chair`, `Vice Chair`, and `Ranking Minority Member`) instead of the JSON names.